x/concentrated-liquidity/lp_test.go

```diff
@@ -0,0 +1,33 @@
+package concentrated_liquidity_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (s *KeeperTestSuite) TestMint() {
+	// testing params
+	// current tick: 85176
+	// lower tick: 84222
+	// upper tick: 86129
+	// liquidity(token in):1517882343751509868544
+	// current sqrt price: 5602277097478614198912276234240
+	// denom0: uosmo
+	// denom1: usdc
+	poolId := uint64(1)
+	currentTick := sdk.NewInt(85176)
+	lowerTick := sdk.NewInt(84222)
+	upperTick := sdk.NewInt(86129)
+	liquidity, ok := sdk.NewIntFromString("1517882343751509868544")
+	s.Require().True(ok)
+	currentSqrtP, ok := sdk.NewIntFromString("5602277097478614198912276234240")
+	s.Require().True(ok)
+	denom0 := "uosmo"
+	denom1 := "usdc"
+
+	s.SetupTest()
+
+	s.App.ConcentratedLiquidityKeeper.CreateNewConcentratedLiquidityPool(s.Ctx, poolId, denom0, denom1, currentSqrtP, currentTick)
+
+	_, _, err := s.App.ConcentratedLiquidityKeeper.Mint(s.Ctx, poolId, s.TestAccs[0], liquidity, lowerTick, upperTick)
```

Comment:
should we be testing the asset0 and asset1 outputs are numbers we expect?

Reply:
We don't have the math for it as of right now